src/mpid/ch4/shm/ipc/cma/cma_post.h

```diff
@@ -15,13 +15,13 @@
     - name        : MPIR_CVAR_CH4_CMA_ENABLE
       category    : CH4
       type        : int
-      default     : 1
+      default     : 0
       class       : none
       verbosity   : MPI_T_VERBOSITY_USER_BASIC
       scope       : MPI_T_SCOPE_ALL_EQ
       description : >-
-        To manually disable CMA set to 0. The environment variable is valid only when the CMA
-        submodule is enabled.
+        Set to 1 to manually enable CMA. It is disabled by default because the CMA requires
+        the ptrace_scope permission, which is often disabled.
```

Comment:
is it possible to check at runtime? i should look at what ucx does...

Reply:
they manually read the ptrace_scope file... https://github.com/openucx/ucx/blob/600b468427546bdfe431dc28147ae2b2916f2a9e/src/uct/sm/scopy/cma/cma_md.c#L53

Reply:
LOL. It's too hacky to my taste. I would rather let system installer deal with this.

Reply:
Yeah. Certainly not worth gating this PR.